test/passport/callbacks.coffee

```diff
@@ -56,19 +56,15 @@ describe 'passport callbacks', ->
     res = { body: { access_token: 'access-token' }, status: 200 }
     @request.end.args[0][0](null, res)
 
-  it 'signs up a user via twitter', (done) ->
+  it 'denies twitter signup', (done) ->
```

Comment:
I also updated a test here for your last PR.

Oookay I'll add CI to this already!